### PR TITLE
Allow injecting of hostAliases in the actionrunner deployment

### PR DESCRIPTION
**Pull request by AngryDeveloper**
This would allow the actionrunner to work with exotic DNS configuration
eg: you have access to some bastion host through its IP but you can't
resolve its name
The config directive is completely optional and won't impact existing
deployments

### CODE REVIEW

**Comment by punkrokk** [comment]
Do we need to also have a docs PR?

**Comment by armab** [comment]
@punkrokk Not for this change.

**Comment by AngryDeveloper** [comment]
This should be better !

File: CHANGELOG.md [hunk 1]
```diff
@@ -3,6 +3,7 @@
 ## In Development
 * Change ingress name from `<release name>-ingress` to <release name>-st2web-ingress, useful when using `stackstorm-ha` as a requirement for another chart. (#112) (by @erenatas)
 * Fix st2web ingress which should have been defined as an Integer instead of a String (#111) (by @erenatas)
+* Add an option to inject hostAliases in the st2actionrunner containers
```
Comment (armab) [inline CHANGELOG.md]:
```suggestion
* Add an option to inject hostAliases in the st2actionrunner containers #114
```

**Pull request merged**